NuRadioMC/simulation/simulation.py

```diff
@@ -562,7 +562,7 @@ def apply_det_response(evt, det, config,
     else:
         dt = 1. / (config['sampling_rate'])
         # start detector simulation
-        efieldToVoltageConverter.run(evt, station, det, channel_ids=channel_ids)  # convolve efield with antenna pattern
+        efieldToVoltageConverter.run(evt, station, det, channel_ids=channel_ids)  # convolve efield with antenna pattern and adds cable delay!
         # downsample trace to internal simulation sampling rate (the efieldToVoltageConverter upsamples the trace to
```

Comment:
If you look at the two doc strings, it seems confusing. eFieldToVoltageConverter: Does it or does it not add the cable delay? We should be very explicit here now.

Reply:
The `efieldToVoltageConverter` does add the cable delay, the `efieldToVoltageConverterPEREFIELD` does not. I know ... confusing.